.env.compose

```
@@ -557,7 +557,46 @@ MCP_WS_TLS=false
+MCP_AUTH_SERVERS=https://auth.gauzy.co
+
+# JWT validation for production (using RS256 with public key)
+MCP_AUTH_JWT_AUDIENCE=https://mcp.gauzy.co
+MCP_AUTH_JWT_ISSUER=https://auth.gauzy.co
+MCP_AUTH_JWT_ALGORITHMS=RS256,ES256
+
+# Option 1: Use JWKS URI for dynamic key discovery (recommended)
+MCP_AUTH_JWT_JWKS_URI=https://auth.gauzy.co/.well-known/jwks.json
+
+# Production authorization server configuration
+MCP_AUTH_SERVERS='[{
```

dotenv-linter (3.3.0)

[warning] [DuplicatedKey] The MCP_AUTH_SERVERS key is duplicated

(DuplicatedKey)